utils/utils.go

```diff
@@ -0,0 +1,7 @@
+package utils
+
+import (
+	"errors"
+)
+
+var JobNotFoundError = errors.New("No job found")
```

Comment:
Maybe move this to an errors package? It will read better if its something like 'errors.JobNotFound' ?